ably/manifest.json

```diff
@@ -0,0 +1,48 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "TBC",
+  "app_id": "ably",
+  "display_on_public_website": true,
+  "tile": {
+    "overview": "README.md#Overview",
+    "configuration": "README.md#Setup",
+    "uninstallation": "README.md#Uninstallation",
+    "support": "README.md#Support",
+    "changelog": "CHANGELOG.md",
+    "description": "Ably's platform powers realtime use cases.",
+    "title": "Ably",
+    "media": [],
```

Comment:
We highly recommend adding images/media for integrations like graphs, etc.

Reply:
Added dashboard screenshot in e9081e8effd1bc5579d24aabe97872f894935de4